source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -202,6 +215,65 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
         }
     }
     
+    func increaseIntrinsicContentSize(_ view: NSView) {
+        let key = String(format: "%p", view)
+        subviewIntrinsicContentSizeCollection[key] = NSValue(size: view.intrinsicContentSize)
+    }
+    
+    func decreaseIntrinsicContentSize(_ view: NSView) {
+        // This Empty function design for override methods
```

Comment:
in decrease and update cases , do we not need to update "subviewIntrinsicContentSizeCollection"  ?

Reply:
yes, you can check this, in subclass override methods